### ISSUE TITLE
Please mark the version in the binaries to downlod

### ISSUE THREAD

**Issue by starbrights** [programminghoch10/Lygisk#25]
I got a reminder that an new Magisk 26 is out. Downloading it - do I download Magisk or Lygisk?
That is not clear.
As of now you can here download latest stable or canary but there is no info about what version it is actually.
Would be great to have this info (and maybe keep one older version too, just in case).
Anyhow, thanks for sharing!

**Comment by binarynoise** [comment]
You download Lygisk.
If unsure, look open the manager, there the version will be left of the install-button.
We probably could change the notification (I assume this is the reminder).
As to keep older versions: just don't delete the manager apk of the old version, if you need to downgrade, just uninstall the manager app, install the old version again and perform a direct install. You should be back at the old version again. Alternatively you can also flash the apk renamed as zip through your recovery to change between versions.

**Comment by kennyke16** [comment]
It's a common sense to add versions to downloadable files.

**Comment by mddvul22** [comment]
I agree.  Not having versions that track Magisk is less than ideal.  I currently have to download the stable version and then compare the file's checksum to my previously installed version to see if it it has changed.
